test/ModAttachr/modAttachr.ts

```diff
@@ -0,0 +1,36 @@
+import { ModAttachr } from "../../src/ModAttachr";
+
+import { mochaLoader } from "../main";
+
+import { EventNames } from "../../src/EventNames";
+
+import {
+    IMod
+} from "../../src/IModAttachr";
+
+mochaLoader.it("Creates a mod and fires the contained events", (): void => {
+    // Arrange
+    const eventNames: EventNames = new EventNames();
+    const dummyMod: IMod[] = [{
+        name: "Dummy Mod",
+        events: {
+            onModDisable: (): string => {
+                return "success";
+            },
+            onModEnable: (): string => {
+                return "success";
+            }
+        },
+        enabled: false
+    }];
+    const modAttachr: ModAttachr = new ModAttachr({
+        mods: dummyMod,
+        storeLocally: true,
```

Comment:
We definitely do *not* want to `storeLocally`. Unit tests shouldn't modify things like `localStorage`.